hcn/hcnloadbalancer.go

```diff
@@ -35,7 +35,7 @@ var (
 	// LoadBalancerFlagsNone is the default.
 	LoadBalancerFlagsNone LoadBalancerFlags = 0
 	// LoadBalancerFlagsDSR enables Direct Server Return (DSR)
-	LoadBalancerFlagsDSR LoadBalancerFlags = 1
+	LoadBalancerFlagsDSR  LoadBalancerFlags = 1
```

Comment:
Is this space addition automated? If not is it necessary to make the change?

Reply:
ah, yes, my IDE must've run formatting tool on this file, because there were some changes.